spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/ZuulMetricsInitializer.java

```diff
@@ -0,0 +1,33 @@
+package org.springframework.cloud.netflix.zuul;
+
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+
+import com.netflix.zuul.monitoring.CounterFactory;
+import com.netflix.zuul.monitoring.TracerFactory;
+
+import lombok.extern.apachecommons.CommonsLog;
+
+@CommonsLog
+public class ZuulMetricsInitializer implements ServletContextListener {
```

Comment:
I don't understand how it relates to filters initialization, so I decided to put it into separate class. I can move it back, but `ZuulFilterInitializer` will know about `CounterFacatory` and `TracerFactory`, I don't think this is actually a good idea. What do you think?